### PR TITLE
MNT/FIX: add detailed_tree from typhos and soft-link attenuator screens

### LINKED ISSUE #932: Ensure custom attenuator screens for AT2K2, AT1K4, etc. load by default
## Expected Behavior
* Loading AT2K2 in typhos, one should see the custom screens we've made by default

## Current Behavior
* Currently, you have to click on `detailed_tree` to get the correct screen
* The default screen is nearly useless and confuses users

## Possible Solution
Rename screens so that typhos's internal mechanism picks the right one, no matter the display type?
Or make a simpler embedded screen?

## Steps to Reproduce (for bugs)
Load attenuator screen in LUCID

## Context
Users indicate this may have changed at some point, but I'm not sure that's the case as I don't think the method has changed in typhos or here.

## Your Environment
pcds-5.1.1

### PR DESCRIPTION
**Pull request by klauer**
## Description
Load correct screens without additional clicks for the solid attenuators

## Motivation and Context
Closes #932 as reported by users

## How Has This Been Tested?
Interactive testing with dev typhos

## Where Has This Been Documented?
PR text + issue

## Pre-merge checklist
- [x] Code works interactively
- [x] Code contains descriptive docstrings, including context and API
- [x] New/changed functions and methods are covered in the test suite where possible
- [x] Test suite passes locally
- [x] Test suite passes on travis
- [x] Ran docs/pre-release-notes.sh and created a pre-release documentation page
- [x] Pre-release docs include context, functional descriptions, and contributors as appropriate

### CODE REVIEW

**Comment by klauer** [comment]
@tangkong - I wanted to draw your attention to this directory in pcdsdevices - `pcdsdevices.ui` contains some custom typhos templates for our devices. These are mostly hand-crafted displays that are intended to override the auto-generated ones, and do so by class name matching with `.ui` filenames.

**Review by tangkong — APPROVED** [review]
Looks good to me, thanks for bringing to my attention.   The changes look pretty straightforward.  

I still need to get setup to test typhos / lucid changes locally, but I don't want to hold up this PR.

File: docs/source/upcoming_release_notes/932-atten_screens.rst [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+- N/A
+
+Bugfixes
+--------
+- N/A
+
+Maintenance
+-----------
+- ``detailed_tree.ui`` was vendored from typhos. The default attenuator screens
+- AT2L0, AT1K4, and AT2K2 will now default to ``detailed_tree.ui``.
```
Comment (ZLLentz) [inline docs/source/upcoming_release_notes/932-atten_screens.rst:27]:
```suggestion
  AT2L0, AT1K4, and AT2K2 will now default to ``detailed_tree.ui``.
```

**Review by ZLLentz — APPROVED** [review]
LGTM, I'll go click around to verify that the dev looks normal to me

**Comment by ZLLentz** [comment]
Looks good to me on the dev, it might even be time to deactivate the `at*_calc` entries in happi.

**Comment by klauer** [comment]
Yeah, those `_calc` entries are probably no longer necessary.

**Pull request merged**